frontend/nextjs/components/ResearchBlocks/ChatResponse.tsx

```
@@ -0,0 +1,119 @@
+                width={20}
+                height={20}
+                className="cursor-pointer text-white"
+              />
+            </button>
+          </div>
+          
+          <div className="flex flex-wrap content-center items-center gap-[15px] pl-5 pr-5">
+            <div className="w-full whitespace-pre-wrap text-base font-light leading-[152.5%] text-white log-message">
+              <div 
+                className="markdown-content prose prose-invert max-w-none"
+                dangerouslySetInnerHTML={{ __html: htmlContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)